src/adlist.h

```diff
@@ -88,6 +88,13 @@ void listRewindTail(list *list, listIter *li);
 void listRotateTailToHead(list *list);
 void listRotateHeadToTail(list *list);
 void listJoin(list *l, list *o);
+static inline void listInitNode(listNode *node, void *value) {
```

Comment:
any strong reason this should be inlined as apposed to all other list functions? this will only effect client creation time and not during client write correct?

Reply:
will do